static/app/components/scoreCard.tsx

```diff
@@ -92,7 +92,7 @@ const HeaderWrapper = styled('div')`
 export const ScoreWrapper = styled('div')`
   display: flex;
   flex-direction: row;
-  align-items: flex-end;
+  align-items: baseline;
```

Comment:
Why this works: the baseline of text containers are the typographic baseline of the text inside.